.github/workflows/php-lint-test.yml

```diff
@@ -5,7 +5,7 @@ on:
 
 env:
   WP_VERSION:        latest
-  WC_VERSION: 5.7.0  # the min supported version as per L-2 policy
+  WC_VERSION: 6.0.0  # the min supported version as per L-2 policy
```

Comment:
Can you confirm this change? From what I understand the minimum L-2 version would be `4.9.0`. Since the L-2 [policy](https://developer.woocommerce.com/2021/05/12/woocommerce-payments-is-adopting-a-new-version-support-policy/) [says](https://developer.woocommerce.com/2020/05/08/wordpress-core-support-policy/) that we will support the current version as well as the latest release from the two previous release branches.

Reply:
> Can you confirm this change? From what I understand the minimum L-2 version would be 4.9.0

Can you elaborate on why `4.9.0`?

`5.7.0` didn't make sense to me. WP is at `5.9` right now, this was likely mistaken as WP_VERSION, not WC.

If it were WP, it should be `5.7`, because we must support `5.7`, `5.8` and latest.

> An L-2 version support policy means supporting the latest version and the 2 versions before it. 
paJDYF-1BP-p2

So the "# the min supported version as per L-2 policy" should be 6.0.0, because as per L-2 we should support 6.0.0, 6.1.1 and 6.2.0 (latest).

Any reason

Reply:
> So the "# the min supported version as per L-2 policy" should be 6.0.0, because as per L-2 we should support 6.0.0, 6.1.1 and 6.2.0 (latest).

Gotcha. Thanks for the explanation 📝